trulens_eval/trulens_eval/feedback/provider/base.py

```diff
@@ -352,7 +353,7 @@ def qs_relevance(self, question: str, context: str) -> float:
         return self.context_relevance(question, context)
 
     def context_relevance_with_cot_reasons(self, question: str,
-                                           context: str) -> Tuple[float, Dict]:
+                                           context: str, temperature: float = 0.0) -> Tuple[float, Dict]:
```

Comment:
should we allow temperature parameter for all feedbacks?

Reply:
IMO I think we should - but only after we've confirmed temperature scaling is substantial enough comparing to prompts iteration. 

Temp scaling has, from what I observed, marginal impact on overall calibration, and I'll validate in the next PR / line of work how effective it is across different score ranges.